apiserver/tools.go

```diff
@@ -60,8 +161,8 @@ func (t *toolsReadCloser) Close() error {
 
 // toolsHandler handles tool upload through HTTPS in the API server.
 type toolsUploadHandler struct {
```

Comment:
I wish we would rename this and other internal tools artefacts to "agentBinaryXXX" etc
Maybe another PR

Reply:
I have adjusted the comment to better signal this. I am still reluctant to change this part because it specifically relates to the interface of the apiserver and that is still attached to the tools wording.